confignode/src/main/java/org/apache/iotdb/confignode/physical/PhysicalPlan.java

```diff
@@ -108,7 +108,7 @@ public static PhysicalPlan create(ByteBuffer buffer) throws IOException {
           plan = new GetOrCreateSchemaPartitionPlan(PhysicalPlanType.GetSchemaPartition);
           break;
         case CreateSchemaPartition:
-          plan = new CreateSchemaPartitionPlan(PhysicalPlanType.CreateSchemaPartition);
+          plan = new CreateSchemaPartitionPlan();
```

Comment:
should be GetSchemaPartitionPlan?

Reply:
I distinguish the operations between query and write about PartitionTable in ConfigNode. When DataNode send a getOrCreate request, ConfigNode first check if there exists Partitions that haven't been assigned. And ConfigNode will use create plan to allocate these non-assigned Partitions.